oss-cloud-backend/handler.js

```diff
@@ -7,33 +7,27 @@ module.exports.getContributors = async (event) => {
   try {
     const {
       sortBy, sortDesc, page, itemsPerPage, searchParam, showHidden,
-    } = event.queryStringParameters;
+    } = event.queryStringParameters || {};
+
     const [contributors, [contributorsLength]] = await databaseService.getContributorsPaging({
-      sortBy,
+      sortBy: typeof (sortBy) === 'string' ? sortBy : 'username',
       sortDesc: sortDesc === 'true',
-      page: parseInt(page, 10),
-      itemsPerPage: parseInt(itemsPerPage, 10),
-      searchParam,
+      page: page ? parseInt(page, 10) : 1,
```

Comment:
I think that default value for page offset needs to be 0, let's check it. I think that you will probably get the second page if you set value to be 1.

Reply:
In databaseService.js, we do `params.page - 1` before putting it into the query.